Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -128,6 +128,30 @@ public extension Node where Context: HTMLLinkableContext {
     }
 }
 
+public extension Attribute where Context: HTMLLinkableContext {
+    /// Assign a hreflang string to this element.
+    /// - parameter hreflang: The hreflang of the link to assign.
+    static func hreflang(_ hreflang: String) -> Attribute {
+        Attribute(name: "hreflang", value: hreflang)
+    }
+}
+
+public extension Attribute where Context: HTMLLinkableContext {
```

Comment:
I think this should be constrained by `Context == HTML.LinkContext`, since it can only be used on `<link>` elements.

Reply:
Updated.